apps/site-nft/src/components/MainRouter/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { AddAccount } from 'components/AddAccount';
+import { Congratulations } from 'components/AddAccount/Congratulations';
+import { GAuth } from 'components/AddAccount/Store/Providers/Google/gauth';
+import { Claim } from '../Claim';
+import { AuthRoute } from '../AuthRoute';
+import { ProfileBuilder } from '../ProfileBuilder';
+import { HomePage } from '../HomePage';
+
+//
+// Trialing a more well-defined routing system
+// This struct is exported so the keys (paths)
+// can be referenced elsewhere in the project.
+export const SiteMap = {
+  Auth: {
+    claim: Claim,
+    profile: ProfileBuilder,
+  },
+  Public: {
+    gauth: GAuth,
+    addAccount: AddAccount,
+    congrats: Congratulations,
+  },
+  default: HomePage,
+}
+
```

Comment:
So this is an incomplete concept.  I wanted to have a system where we could define all routes in a way that the rest of the site could refer to it (ie - to use the compiler to catch any 404's).  However, it didn't quite work because the Auth path needs a bit more thought.

I originally thought it would be great to have a very black-box system, where the each page would define it's own dependents, but this hasn't worked out: in the end I think it's better to have a single-source-of-truth as to what our site-map looked like.